Tasks/DotNetCoreCLIV2/dotnetcore.ts

```diff
@@ -347,30 +344,16 @@ export class dotNetExe {
             } 
             return resolvedProjectFiles;
         }
-
         return projectFiles;
     }
 
     private isWebSdkUsed(projectfile: string) {
         var fileBuffer: Buffer = fs.readFileSync(projectfile);
-        var fileEncodeType = fileEncoding.detectFileEncoding(projectfile, fileBuffer);
-        var webConfigContent: string = fileBuffer.toString(fileEncodeType[0]);
-
-        if(fileEncodeType[1]) {
-            webConfigContent = webConfigContent.slice(1);
-        }
-
-        var ltxDom = new ltxdomutility.LtxDomUtility(webConfigContent);
-        var projectDomArray: Array<any> = ltxDom.getElementsByTagName("project");
-
-        for(var project of projectDomArray) {
-            if(project.attrs['Sdk'].toLowerCase() == "microsoft.net.sdk.web" || 
-                project.attrs['sdk'].toLowerCase() == "microsoft.net.sdk.web") {
-                return true;
-            }
-        }
+        var webConfigContent: string = fileBuffer.toString();
```

Comment:
Should you put this block in try-catch to ignore any errors while detecting the sdk

Reply:
Done